tests/transfer/test_transfer_key_to_key.py

```diff
@@ -37,6 +44,91 @@ def test_transfer_submit(continuous_market, vega: VegaService, page: Page):
     vega.wait_for_total_catchup()
     expected_confirmation_text = re.compile(r"Transfer completeYour transaction has been confirmed View in block explorerTransferTo .{6}….{6}1\.00 tDAI")
     actual_confirmation_text = page.get_by_test_id('toast-content').text_content()
-    print(f"Actual text is: {actual_confirmation_text}")
     assert expected_confirmation_text.search(actual_confirmation_text), f"Expected pattern not found in {actual_confirmation_text}"
-    
\ No newline at end of file
+    
+
+@pytest.mark.usefixtures("page", "auth", "risk_accepted")
+def test_transfer_vesting_below_minimum(continuous_market, vega: VegaService, page: Page):
+
+    vega.update_network_parameter(
+    "mm", parameter="transfer.minTransferQuantumMultiple", new_value="100000"
+    )
+    vega.wait_for_total_catchup()
+    
+    
+    create_and_faucet_wallet(vega=vega, wallet=PARTY_A, amount=1e3)
+    create_and_faucet_wallet(vega=vega, wallet=PARTY_B, amount=1e5)
+    create_and_faucet_wallet(vega=vega, wallet=PARTY_C, amount=1e5)
+    vega.wait_for_total_catchup()
+
+    asset_id = vega.find_asset_id(symbol="tDAI")
+    next_epoch(vega=vega)
+
+    vega.recurring_transfer(
+        from_key_name=PARTY_A.name,
+        from_account_type=vega_protos.vega.ACCOUNT_TYPE_GENERAL,
+        to_account_type=vega_protos.vega.ACCOUNT_TYPE_REWARD_MAKER_PAID_FEES,
+        asset=asset_id,
+        asset_for_metric=asset_id,
+        metric=vega_protos.vega.DISPATCH_METRIC_MAKER_FEES_PAID,
+        amount=100,
+        factor=1.0,
+    )
+    # Generate trades for non-zero metrics
+    vega.submit_order(
+        trading_key=PARTY_B.name,
+        market_id=continuous_market,
+        order_type="TYPE_LIMIT",
+        time_in_force="TIME_IN_FORCE_GTC",
+        side="SIDE_SELL",
+        price=0.30,
+        volume=100,
+    )
+    vega.submit_order(
+        trading_key=PARTY_C.name,
+        market_id=continuous_market,
+        order_type="TYPE_LIMIT",
+        time_in_force="TIME_IN_FORCE_GTC",
+        side="SIDE_BUY",
+        price=0.30,
+        volume=100,
+    )
+    vega.wait_for_total_catchup()
+    next_epoch(vega=vega)
+    next_epoch(vega=vega)
+    page.goto('/#/portfolio')
+    expect(page.get_by_test_id('transfer-form')).to_be_visible
+    
+    page.get_by_test_id("manage-vega-wallet").click()
```

Comment:
again not addressable, more something that would be good to chat about after the monorepo move but my issue with test code in this fashion is that the detail in it is so granular it is hard to get the abstracted information necessary to 'read' what the test is doing. If I want to re-use this functionality it becomes a copy and paste judgement instead of just a function that will perform the action I want, like populating a form, or submitting an order in the ui etc.